src/containers/Home/index.tsx

```diff
@@ -1,7 +1,63 @@
 import React from 'react';
+import useForm, { isRequired, isEmail, maxChars } from '../../hooks/useForm';
 
 const Home: React.FC = () => {
-  return <h2>Home</h2>;
-}
+  const { fields, onChange, errors, disableForm } = useForm([
+    { field: 'name', value: '', validators: [isRequired] },
+    { field: 'email', value: '', validators: [isRequired, isEmail] },
+    { field: 'comment', value: '', validators: [maxChars(20)] },
+  ]);
+
+  const onSubmitHandler = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+  };
+
+  return (
+    <React.Fragment>
+      <h2>Home</h2>
+      <form onSubmit={onSubmitHandler}>
+        {/* Name field*/}
+        <label>Name: </label>
+        <br />
+        <input
+          type="text"
+          name="name"
+          onChange={onChange}
```

Comment:
All of the inputs should have the same `onChange` ?

Reply:
yes, it is to detect the change of the input value and to be handled by the useForm